<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modificar Clínica</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>721</width>
<<<<<<< Updated upstream:ui/clinicas_editar.ui
      <height>139</height>
=======
      <height>221</height>
>>>>>>> Stashed changes:ui/animal_editar.ui
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <item row="3" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="nameInput"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="speciesLabel">
       <property name="text">
        <string>Especie:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="speciesInput"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Descripción:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QTextEdit" name="descriptionInput">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="ownerLabel">
       <property name="text">
        <string>Dueño:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="ownerInput"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="idDisplayLabel">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
<<<<<<< Updated upstream:ui/clinicas_editar.ui
     <item row="1" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nameInput"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="municipioLabel">
       <property name="text">
        <string>Municipio:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="municipioInput"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="provinciaLabel">
       <property name="text">
        <string>Provincia:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="provinciaInput"/>
     </item>
     <item row="0" column="1">
=======
     <item row="2" column="1">
>>>>>>> Stashed changes:ui/animal_editar.ui
      <widget class="QLabel" name="idDisplay">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
<<<<<<< Updated upstream:ui/clinicas_editar.ui
        <string>ID de la clínica</string>
=======
        <string>ID del animal</string>
>>>>>>> Stashed changes:ui/animal_editar.ui
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="volverButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guardarButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar Cambios</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
